ziggurat-core-crawler/src/connection.rs

```diff
@@ -35,7 +35,7 @@ impl Hash for KnownConnection {
 }
 
 impl KnownConnection {
-    pub fn new(a: IpAddr, b: IpAddr) -> Self {
+    pub fn new(a: SocketAddr, b: SocketAddr) -> Self {
```

Comment:
ah! yes
just a note so we don't forget:
we should check the XRPL crawler more when we port this change there - there was some issue about known_connnections and a default port...

Reply:
thanks, good point!